sequelize/db.js

```diff
@@ -1,69 +1,74 @@
-const { Sequelize, DataTypes } = require('sequelize')
-const logger = require('../lib/log')(__filename)
-require('dotenv').config()
+const { Sequelize, DataTypes } = require("sequelize");
+const logger = require("../lib/log")(__filename);
+require("dotenv").config();
 
-let sequelize
-const dbModule = {}
+let sequelize;
+const dbModule = {};
 
-const models = {}
+const models = {};
 
 dbModule.start = async () => {
-  logger.info('starting sequelize server')
+  logger.info("starting sequelize server");
   try {
-    sequelize = new Sequelize(process.env.DATABASE, process.env.PG_USER, process.env.PASSWORD, {
-      host: process.env.HOST,
-      dialect: 'postgres',
-      pool: {
-        max: 5,
-        min: 0,
-        idle: 10000
+    sequelize = new Sequelize(
+      process.env.DATABASE,
+      process.env.PG_USER,
+      process.env.PASSWORD,
+      {
+        host: process.env.HOST,
+        dialect: "postgres",
+        pool: {
+          max: 5,
+          min: 0,
+          idle: 10000,
+        },
       }
-    })
-    models.Accounts = await sequelize.define('account', {
+    );
+    models.Accounts = await sequelize.define("account", {
       username: {
         type: DataTypes.STRING,
         unique: true,
-        allowNull: false
+        allowNull: true,
```

Comment:
Is it possible to just altogether remove `username` as an attribute?
If we are no longer using `username` and it will not have any significance, then I think we should remove the `username` attribute.

Reply:
We are gonna discuss about it soon